SeleniumPactice/April2022SeleniumSession/src/main/java/seleniumsessions/CustomizedWait.java

```diff
@@ -0,0 +1,20 @@
+package seleniumsessions;
+
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.chrome.ChromeDriver;
+
+import io.github.bonigarcia.wdm.WebDriverManager;
```

Comment:
@VinithaDar Do we need to import this here?

Reply:
yes, we need for webdrivermanager method.